packages/config/config/devices/0x0131/ne-neo-ds07z.json

```
@@ -0,0 +1,124 @@
+{
+	"manufacturer": "Zipato",
+	"manufacturerId": "0x0131",
+	"label": "NE-NEO-DS07Z",
```

```suggestion
	"label": "AH-NEO-DS07Z",
```
probably? If so, also rename the file.

---

ditto